src/__tests__/integration.ts

```diff
@@ -230,28 +230,32 @@ describe('graphql server', () => {
     let charlieOrg: OrganizationType
 
     beforeEach(async () => {
-      let emptyOrg = await organizations.addOrganization(user, 'Alpha Club', OrgType.localClimbingOrganization)
       let document: any = {
+        displayName: 'Alpha Club',
         email: 'admin@alpha.com',
         associatedAreaIds: [ca.metadata.area_id, wa.metadata.area_id]
       }
-      alphaOrg = await organizations.updateOrganization(user, emptyOrg.orgId, document)
+      alphaOrg = await organizations.addOrganization(user, OrgType.localClimbingOrganization, document)
         .then((res: OrganizationType | null) => {
           if (res === null) throw new Error('Failure mocking organization.')
           return res
         })
 
-      emptyOrg = await organizations.addOrganization(user, 'Beta Club', OrgType.localClimbingOrganization)
-      document = { email: 'admin@beta.com' }
-      betaOrg = await organizations.updateOrganization(user, emptyOrg.orgId, document)
+      document = {
+        displayName: 'Beta Club',
+        email: 'admin@beta.com'
```

Comment:
isn't it good practice to avoid using domains we don't own? alpha.com in the above example is  a registered domain.  Unlikely to be a problem since we aren't sending emails now, but might be in future?

Reply:
in the future  what about test@example.com or test@openbeta.io?  the later is actually valid.  It goes to a catch-all mailbox.